extensions/objects/models/objects_test.py

```diff
@@ -60,9 +60,9 @@ def test_boolean_validation(self):
         """Tests objects of type Boolean."""
         mappings = [('', False), (False, False), (True, True), (None, False)]
         invalid_values_with_error_messages = [
-            ({}, r'Expected bool, received \{\}'),
-            ([], r'Expected bool, received \[\]'),
-            (['a'], r'Expected bool, received \[u\'a\'\]'),
+            ({}, re.escape('Expected bool, received {}')),
+            ([], re.escape('Expected bool, received []')),
+            (['a'], re.escape('Expected bool, received [\'a\']')),
```

Comment:
Why do we need re.escape here and not in the line below? (Similar questions throughout this file.)

In fact, do we even need the "r" prefix? Not every message below has it.

Reply:
Because, some strings contain chars that have different meaning in regexes. I have removed the `r'` from some places.